MyOwnORM/MyOwnORM/CustomDbSet.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustomDbSet<T> : CustomDbSetRepository<T> where T : class
```

Comment:
why do you need this class

Reply:
Because I wanted to create ORM very similar to EF and this class serves to register entities and manipulating them using 'CustomDbSetRepository'

Reply:
Yes, it should be similar to EF, but now I can see that this and some more classes exist without any mean. If you want to extend it in future, ok but empty objects without definition should not exist

Reply:
Alright

Reply:
Done

Reply:
you haven't deleted it. And it still doesn't do anything